src/esp/bindings/AttributesBindings.cpp

```diff
@@ -43,9 +44,14 @@ void initAttributesBindings(py::module& m) {
   // NOLINTNEXTLINE(bugprone-unused-raii)
   py::class_<AbstractManagedObject, AbstractManagedObject::ptr>(
       m, "AbstractManagedObject");
+  // ==== AbstractFileBasedManagedObject ====
+  // NOLINTNEXTLINE(bugprone-unused-raii)
+  py::class_<AbstractFileBasedManagedObject, esp::core::AbstractManagedObject,
+             AbstractFileBasedManagedObject::ptr>(
+      m, "AbstractFileBasedManagedObject");
 
   // ==== AbstractAttributes ====
-  py::class_<AbstractAttributes, esp::core::AbstractManagedObject,
+  py::class_<AbstractAttributes, esp::core::AbstractFileBasedManagedObject,
```

Comment:
Likewise for here. Why are you trying so hard to hide the inheritance from Python?

Reply:
Not trying hard at all :).  As I understood it when I first coded this last summer, it could be done either way.  I saw no benefit exposing them to python.

Reply:
Ah nvm, just confused because one is bound with the namespace and the other is not, which is a bit odd.

Reply:
Would there be some benefit that I am overlooking, other than getting rid of the 2nd NoLintNextLine?  If you think this would be a good change, I'm certainly open to it. :)